components/config-provider/index.en-US.md

```diff
@@ -76,6 +77,19 @@ ConfigProvider.config({
 });
 ```
 
+### ConfigProvider.useConfig()
+
+Get the value of the parent `Provider`. Such as `DisabledContextProvider`, `SizeContextProvider`.
```

Comment:
```suggestion
Available since `5.2.0`. Get the value of the parent `Provider`. Such as `DisabledContextProvider`, `SizeContextProvider`.
```
中文文档同理

Reply:
done，文档里加上了可用版本